XBeeLibrary.Xamarin/Connection/Bluetooth/BluetoothInterface.cs

```diff
@@ -174,6 +181,10 @@ public void Open()
 					if (!IsOpen)
 						throw new Exception();
 
+					// Request a larger MTU.
+					mtu = await device.RequestMtuAsync(REQUESTED_MTU);
+					Console.WriteLine("----- MTU: " + mtu);
```

Comment:
Do we need this line?

Reply:
Well, it's just debug information, it does not appear in release builds. We have other lines like this in the different operations (open, close, write, read...).